### PR TITLE
fix: updated dependency + removed ip

### PR DESCRIPTION
**Pull request by eddiejaoude**
## Fixes Issue

```
➜  LinkFree git:(main) ✗ npm outdated
Package                      Current   Wanted   Latest  Location                                  Depended by
@babel/core                   7.22.5   7.22.9   7.22.9  node_modules/@babel/core                  LinkFree
@next/mdx                     13.4.7  13.4.12  13.4.12  node_modules/@next/mdx                    LinkFree
@playwright/test              1.35.1   1.36.1   1.36.1  node_modules/@playwright/test             LinkFree
@storybook/addon-a11y         7.0.23    7.1.0    7.1.0  node_modules/@storybook/addon-a11y        LinkFree
@storybook/addon-essentials   7.0.23    7.1.0    7.1.0  node_modules/@storybook/addon-essentials  LinkFree
@storybook/nextjs             7.0.23    7.1.0    7.1.0  node_modules/@storybook/nextjs            LinkFree
@storybook/react              7.0.23    7.1.0    7.1.0  node_modules/@storybook/react             LinkFree
@storybook/theming            7.0.23    7.1.0    7.1.0  node_modules/@storybook/theming           LinkFree
@tailwindcss/forms             0.5.3    0.5.4    0.5.4  node_modules/@tailwindcss/forms           LinkFree
babel-loader                   9.1.2    9.1.3    9.1.3  node_modules/babel-loader                 LinkFree
eslint                        8.43.0   8.45.0   8.45.0  node_modules/eslint                       LinkFree
eslint-config-next            13.4.7  13.4.12  13.4.12  node_modules/eslint-config-next           LinkFree
eslint-plugin-storybook       0.6.12   0.6.13   0.6.13  node_modules/eslint-plugin-storybook      LinkFree
mongoose                       7.3.3    7.4.0    7.4.0  node_modules/mongoose                     LinkFree
next                          13.4.7  13.4.12  13.4.12  node_modules/next                         LinkFree
next-auth                     4.22.1   4.22.3   4.22.3  node_modules/next-auth                    LinkFree
pino-pretty                   10.0.0   10.1.0   10.1.0  node_modules/pino-pretty                  LinkFree
postcss                       8.4.24   8.4.27   8.4.27  node_modules/postcss                      LinkFree
prettier                       2.8.8    2.8.8    3.0.0  node_modules/prettier                     LinkFree
storybook                     7.0.23    7.1.0    7.1.0  node_modules/storybook                    LinkFree
tailwindcss                    3.3.2    3.3.3    3.3.3  node_modules/tailwindcss                  LinkFree
```

to

```
➜  LinkFree git:(main) ✗ npm outdated
Package   Current  Wanted  Latest  Location               Depended by
prettier    2.8.8   2.8.8   3.0.0  node_modules/prettier  LinkFree
```

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8337"></a>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,6 @@
     "remark": "^14.0.3",
     "remark-gfm": "^3.0.1",
     "remark-prism": "^1.3.6",
```
Comment (dan-mba) [inline package.json:40]: The updates to dependencies aren't here, only the removal of `request-ip`

Comment (eddiejaoude) [reply package.json:40]: OMG really 😱 thank you!

Comment (eddiejaoude) [reply package.json:40]: The package lock file has been updated with the latest changes https://github.com/EddieHubCommunity/LinkFree/pull/8337/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519

Comment (eddiejaoude) [reply package.json:40]: oh yes that is confusing! I will do that now, thank you 👍

**Pull request merged**